content/3_Silver/Graph_Traversal.mdx

```diff
@@ -407,15 +438,22 @@ public static void main(String[] args) throws IOException {
 <CPPSection>
 
 ```cpp
-const int MAX_N = 1e6;
+int n = 6;
+vector<int> adj[n];
+bool visited[n];
```

Comment:
s. a.

Reply:
this is not fixed yet - VLA is not allowed in c++

Reply:
yepp im aware -- fixing it now